perf.groovy

```diff
@@ -341,15 +408,18 @@ def static getOSGroup(def os) {
                 }
             }
             def osGroup = getOSGroup(os)
-            def architecture = 'x64'
-            def configuration = 'Release'
             def runType = isPR ? 'private' : 'rolling'
             def benchViewName = isPR ? 'coreclr private \$BenchviewCommitName' : 'coreclr rolling \$GIT_BRANCH_WITHOUT_ORIGIN \$GIT_COMMIT'
 
             steps {
                 shell("bash ./tests/scripts/perf-prep.sh --throughput")
```

Comment:
Ah, good call, I missed that it was also doing a build and not just downloading bits. It looks like it's already grabbing artifacts on perf, so I just need to modify the throughput flow to archive/download `_/fx/**`, right?